docs/samples/browser-plugin-meetings/app.js

```diff
@@ -2823,6 +2827,17 @@ const participantsList = document.querySelector('#participant-list');
 const participantTable = document.querySelector('#participant-table');
 const participantButtons = document.querySelector('#participant-btn');
 
+participantTable.addEventListener('click', (event) => {
+  if (event.target.type === 'radio') {
+    const selectedParticipant = meeting.members.membersCollection.get(event.target.id);
+    if (!selectedParticipant) {
+      return;
+    }
+    const muteButton = document.getElementById('mute-participant-btn')
+    muteButton.innerText = selectedParticipant.isAudioMuted ? 'Request to unmute': 'Mute';
```

Comment:
FYI, depending on meeting configuration, it is sometimes possible to actually unmute someone not just request to unmute, so the text "request to unmute" is not entirely correct, but I guess that can be improved further in another PR, not a blocker for this one

Reply:
taken care now